charts/workspacemanager/templates/deployment.yaml

```diff
@@ -87,7 +89,37 @@ spec:
         - name: cloudsql-sa-creds
           mountPath: /secrets/cloudsql
           readOnly: true
+      {{- if .Values.enableProxy }}
+      - name: oidc-proxy
+        image: "{{ .Values.proxyImage.repository }}:{{ .Values.proxyImage.version }}"
+        ports:
+        - containerPort: 443
+        - containerPort: 80
+        volumeMounts:
+        - mountPath: /etc/apache2/sites-available/site.conf
```

Comment:
Recommend using a descriptive name vs. the generic `site.conf` here. The sites-available and `sites-enabled` pattern allows for `0..N` entries in these directories. However, it's still better for human debugging and potential automation uses to use a standard naming pattern prefix (or name) for these types of conf files.

Reply:
That's actually good to know - I don't think we'd ever use more than one entry. Should we be using a different pattern (Google points me to using `httpd.conf`)? Just renaming this one in place seemed to cause issues with the proxy not reading the config.